src/exercise/01.md

```diff
@@ -68,4 +68,4 @@ using JavaScript as well.
 ## 🦉 Feedback
 
 Fill out
-[the feedback form](https://ws.kcd.im/?ws=React%20Fundamentals%20%E2%9A%9B&e=01%3A%20Basic%20JavaScript-rendered%20Hello%20World&em=).
+[the feedback form](https://ws.kcd.im/?ws=React%20Fundamentals%20%E2%9A%9B&e=01%3A%20Basic%20JavaScript-rendered%20Hello%20World&em=jonathan%40alley.co).
```

Comment:
You *might* want to remove your email address here; as this this a public repo, the information is technically publicaly available. This is due to your entry of the email address in `npm run setup`